packages/react-composites/review/beta/react-composites.api.md

```diff
@@ -1140,9 +1152,19 @@ export interface ChatAdapterThreadManagement {
     downloadAttachment: (options: {
         attachmentUrl: string;
     }) => Promise<AttachmentDownloadResult>;
+    downloadResourceToCache(resourceDetails: {
+        threadId: string;
+        messageId: string;
+        resourceUrl: string;
+    }): void;
     fetchInitialData(): Promise<void>;
     loadPreviousChatMessages(messagesToLoad: number): Promise<boolean>;
     removeParticipant(userId: string): Promise<void>;
+    removeResourceFromCache(resourceDetails: {
```

Comment:
Would be good to export this as a type for reuse

Reply:
Done